test/test_fully_connected_layer.h

```diff
@@ -69,18 +74,19 @@ TEST(fully_connected, train) {
 
 TEST(fully_connected, train2) {
     network<sequential> nn;
-    gradient_descent optimizer;
+    gradient_descent    optimizer;
 
-    nn << fully_connected_layer<tan_h>(4, 6)
-       << fully_connected_layer<tan_h>(6, 3);
+    nn << fully_connected_layer<tan_h>(4, 6) << fully_connected_layer<tan_h>(6, 3);
```

Comment:
Please add a line break between `fully_connected_layer<tan_h>(4, 6)` and `fully_connected_layer<tan_h>(6, 3)`.